lib/cib/cib_ops.c

```diff
@@ -399,8 +399,8 @@ update_cib_object(xmlNode * parent, xmlNode * update)
     CRM_CHECK(object_name != NULL, return -EINVAL);
 
     object_id = ID(update);
-    crm_trace("Processing <%s id='%s'>",
-              pcmk__s(object_name, "<null>"), pcmk__s(object_id, "<null>"));
+    crm_trace("Processing update for <%s%s%s>", object_name,
+              ((object_id == NULL)? "" : " id="), pcmk__s(object_id, ""));
```

Comment:
'' around the potentially empty id - as before - might still be less confusing even if we have <> as delimiter.

Reply:
In this case, a NULL id is actually for an element that doesn't take an ID, so we don't want id='', but it still makes sense to quote id when it is used